server.js

```diff
@@ -147,36 +137,28 @@ app.put("/api/profile/:profile_id/projects/:_id",function update(req, res) {
   });
 });
 
-// End Point 8 - Add New vacation under planning
-
 app.post('/api/profile/:profile_id/vacation',function(req,res){
-  console.log(JSON.stringify(req.body));
   db.Profile.findById(req.params.profile_id,function(err,foundProfile){
+    var newVacation=new db.Vacation(req.body);
 
-  var newVacation=new db.Vacation(req.body);
-
-  foundProfile.vacation.push(newVacation);
-  foundProfile.save(function(err,savedprofile){
-    res.json(savedprofile);
-  })
-})
+    foundProfile.vacation.push(newVacation);
+    foundProfile.save(function(err,savedProfile){
+      res.json(savedProfile);
+    });
+  });
 });
 
-//End Point 9 - Get All Holiday destinations
-
 app.get('/api/profile/:profile_id/vacation',function(req,res){
-  db.Profile.find({},function(err,vacation){
```

Comment:
This does NOT get all vacations. It gets all profiles. Your callback argument should be `profiles`, not `vacation`.